integration/s3client_test.go

```diff
@@ -110,4 +110,17 @@ var _ = Describe("S3client", func() {
 			Ω(err).ShouldNot(HaveOccurred())
 		})
 	})
+
+	Context("when using a roleArn", func() {
+		BeforeEach(func() {
+			if len(os.Getenv("S3_TESTING_AWS_ROLE_ARN")) == 0 || len(os.Getenv("S3_ROLE_RESTRICTED_TESTING_BUCKET")) == 0 {
```

Comment:
Maybe slightly clearer not to use len() here, since the code isn't doing anything with character positions.
```suggestion
			if os.Getenv("S3_TESTING_AWS_ROLE_ARN") == "" || os.Getenv("S3_ROLE_RESTRICTED_TESTING_BUCKET") == "" {
```